pkg/operator/ceph/cluster/controller.go

```diff
@@ -322,6 +324,20 @@ func (c *ClusterController) onAdd(obj interface{}) {
 	objectStoreUserController := objectuser.NewObjectStoreUserController(c.context, cluster.Namespace, cluster.ownerRef)
 	objectStoreUserController.StartWatch(cluster.stopCh)
 
+	// Start the object bucket provisioner
+	bucketProvisioner := bucket.NewProvisioner(c.context.Clientset, cluster.Namespace)
+	config, err := rest.InClusterConfig()
```

Comment:
Very open to re-using Config set earlier in rook-ceph but was not able to find it.

Reply:
Let us know when we can use this change.

Reply:
@jeffvance This is merged now.